client/src/components/pages/Players.js

```diff
@@ -319,180 +333,174 @@ class Players extends Component {
     };
 
     for (const [index, player] of players.entries()) {
-      await this.handlePlayerEdit({ seedName: getSeedName(index), seedNum: index + 1 }, player._id);
-      this.setState({
-        refreshPercent: Math.min(100, Math.round((100 * (index + 1)) / this.state.players.length)),
-      });
+      await handlePlayerEdit({ seedName: getSeedName(index), seedNum: index + 1 }, player._id);
+
+      setRefreshPercent(Math.min(100, Math.round((100 * (index + 1)) / players.length)));
     }
   };
 
-  render() {
-    return (
-      <Content className="content">
-        <div className="Players-topbar">
-          <Menu
-            mode="horizontal"
-            className="Players-mode-select"
-            onClick={this.handleModeChange}
-            selectedKeys={[this.state.mode]}
-          >
-            <Menu.Item key="players">Players ({this.state.players.length})</Menu.Item>
-            {this.state.hasTeams && (
-              <Menu.Item key="teams">Teams ({this.state.teams.length})</Menu.Item>
+  return (
+    <Content className="content">
+      <div className="Players-topbar">
+        <Menu
+          mode="horizontal"
+          className="Players-mode-select"
+          onClick={handleModeChange}
+          selectedKeys={[mode]}
+        >
+          <Menu.Item key="players">Players ({players.length})</Menu.Item>
+          {hasTeams && <Menu.Item key="teams">Teams ({teams.length})</Menu.Item>}
+        </Menu>
+
+        <div>
+          <span className="Players-sort">Sort by:</span>
+          <Radio.Group value={sort} onChange={handleSortChange}>
+            {mode === "players" ? (
+              <>
+                <Radio.Button value="rank">Rank</Radio.Button>
+                <Radio.Button value="alpha">Alphabetical</Radio.Button>
+                {!hasPlayerSeeds() && <Radio.Button value="seed">Seed</Radio.Button>}
+                {!hasTeams && hasGroups && <Radio.Button value="group">Group</Radio.Button>}
+                <Radio.Button value="country">Country</Radio.Button>
+                <Radio.Button value="reg">Reg Time</Radio.Button>
+              </>
+            ) : (
+              <>
+                <Radio.Button value="alpha">Alphabetical</Radio.Button>
+                {hasTeamSeeds() && <Radio.Button value="seed">Seed</Radio.Button>}
+                <Radio.Button value="group">Group</Radio.Button>
+                <Radio.Button value="rank">Avg Rank</Radio.Button>
+              </>
             )}
-          </Menu>
-
+          </Radio.Group>
+        </div>
+      </div>
+
+      {mode === "teams" && isAdmin() && (
+        <Collapse>
+          <Panel header={`Add new team`} key="1">
+            Type all player names separated by commas, with the captain's name first.
```

Comment:
now that you don't need to type commas anymore, you could change the text to something like this
> Add all player names, with the captain's name first